src/img/lists/arrow.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg" width="40" height="64" viewBox="0 0 40 64"><title>arrow-right</title><path d="M38.524 28.302L11.968 1.561C10.989.52 9.768 0 8.3 0 6.834 0 5.613.52 4.635 1.56L1.539 4.643C.534 5.655.032 6.887.032 8.34c0 1.424.502 2.67 1.507 3.738L21.335 32 1.539 51.964C.534 52.977.032 54.209.032 55.661c0 1.424.502 2.67 1.507 3.738l3.096 3.081C5.639 63.493 6.862 64 8.3 64c1.44 0 2.661-.506 3.666-1.52l26.557-26.742c1.005-1.068 1.508-2.314 1.508-3.738 0-1.452-.503-2.684-1.508-3.698z"/></svg>
```

Comment:
I think for now, let's remove the svgs from version control since we'll wait to see how icons will be handled.